app/scripts/modules/core/src/naming/naming.service.ts

```diff
@@ -69,14 +70,12 @@ export class NamingService {
     return split.join('-');
   }
 
-  public getSequence(serverGroupName: string): string {
-    const split = serverGroupName.split('-'),
-      isVersioned = NamingService.VERSION_PATTERN.test(split[split.length - 1]);
-
-    if (isVersioned) {
-      return split.pop();
+  public getSequence(monikerSequence: number): string {
+    if (monikerSequence == null) {
```

Comment:
This is a little tricky. You want to return "N/A" if `monikerSequence` is `null` or `undefined`. This works because of the `==`, but I think our linter will complain that you're not using `===`.

lodash has a nice method to handle that: [isNil](https://lodash.com/docs/4.17.4#isNil), so you could write this as:
```javascript
if (isNil(monikerSequence)) {
```
and still satisfy the linter (and not have to worry about whether the value is `null` or `undefined`)

Reply:
I'll go ahead and make this change.  Thanks @anotherchrisberry